elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -280,12 +286,14 @@ public ElideResponse delete(
             } else {
                 doc = new JsonApiDocument();
             }
-            RequestScope requestScope = new RequestScope(doc, transaction, user, dictionary, mapper, securityMode);
+            RequestScope requestScope = new RequestScope(
+                    doc, transaction, user, dictionary, mapper, auditLogger, securityMode);
             DeleteVisitor visitor = new DeleteVisitor(requestScope);
             Supplier<Pair<Integer, JsonNode>> responder = visitor.visit(parse(path));
             requestScope.runDeferredPermissionChecks();
             transaction.flush();
             ElideResponse response = buildResponse(responder.get());
+            auditLogger.commit();
```

Comment:
Well, that's the name in the interface – and other things are already using that name in other projects. I'm somewhat reluctant to make a breaking change like that. If we did decide to change it, what do you think about "save"? @aklish what do you think?